docs/user/rstudio/ide/guide/tools/terminal.qmd

```diff
@@ -2,93 +2,100 @@
 
 ## Getting Started
 
 The **Terminal tab** is next to the **Console tab**. Click on the Terminal tab and a new terminal session will be created (if there isn't one already).
 
--   If the tab isn't visible, show it via **Shift+Alt+T** (**Tools-\>Terminal-\>Move Focus to Terminal**).
+-   If the tab isn't visible, show it via **Tools** \> **Terminal** \> **Move Focus to Terminal** (or use the kebyoard shortcut <kbd>Shift+Alt+T</kbd>).
 
--   The full list of terminal-related RStudio commands and keyboard shortcuts is in [Appendix B](https://support.rstudio.com/hc/en-us/articles/115010737148-Using-the-RStudio-Terminal-in-the-RStudio-IDE#appendixb). Most terminal-related commands can be found on the **Terminal dropdown** menu in the terminal pane, and on the **Tools-\>Terminal** menu.
+-   The full list of terminal-related keyboard shortcuts is in the Reference: Keyboard Shortcuts [Terminal section](../../reference/shortcuts.html#terminal). Most terminal-related commands can be found on the **Terminal dropdown** menu in the terminal pane, and on the **Tools-\>Terminal** menu.
 
 Here's a terminal with the output of simple command:
 
 ![](images/terminal-basic.png){fig-alt="screenshot of terminal showing simple output"}
 
-The visual appearance of the terminal is controlled via the **Tools-\>Global Options-\>Appearance** settings. 
+The visual appearance of the terminal is controlled via the **Tools** \> **Global Options** \> **Appearance** settings. 
 
 ## Multiple Terminals
 
-RStudio supports multiple terminal sessions. To start another terminal session, use the **New Terminal** command on the Terminal dropdown menu, or **Alt+Shift+R**.
+RStudio supports multiple terminal sessions. To start another terminal session, use the **New Terminal** command on the Terminal dropdown menu, or <kbd>Alt+Shift+R</kbd>.
 
 ![](images/terminal-menu.png){fig-alt="screenshot showing terminal menu opened"}
+Each terminal session is independent, with its own system shell process and buffer. Switch between them by:
 
-Each terminal session is independent, with its own system shell process and buffer. Switch between them by **(1)** using the arrows next to the drop-down menu, **(2)** clicking on the terminal's name in the drop-down, or **(3)** using the **Previous Terminal**/**Next Terminal** drop-down menu commands. The dropdown button shows the name of the currently selected terminal session ("Terminal 3" in the following screenshot):
+1. Using the arrows next to the drop-down menu  
+2. Clicking on the terminal's name in the drop-down  
+3. Using the **Previous Terminal**/**Next Terminal** drop-down menu commands. The dropdown button shows the name of the currently selected terminal session ("Terminal 1" in the following screenshot):  
```

Comment:
This should be an unordered list if you do not have to complete each item in this order.

Reply:
Resolved with unordered list.